specification/network/resource-manager/Microsoft.Network/stable/2024-07-01/networkSecurityPerimeter.json

```diff
@@ -2273,6 +2315,36 @@
         }
       },
       "description": "Result of the request to list NSP logging configuration. Contains a list of NSP logging configurations and a URL link to get the next set of results."
+    },
+    "NspServiceTagsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NspServiceTagsResource"
+          },
+          "description": "Gets paged list of NSP service tags."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next page of results."
```

Comment:
>NspServiceTagsResource [](http://example.com/codeflow?start=5&length=22)

It seems this data is being added as `additionalProperties` as it is not defined under properties.

Pls follow  model similar to `NspLoggingConfigurationListResult` which should resolve the lintdiff error.